scripts/ci/get_modified_boards.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+# A script to generate a list of boards that have changed or added and create an
+# arguemnts file for sanitycheck to allow running more tests for those boards.
+
+import sys
+import re, os
+from email.utils import parseaddr
+import sh
+import logging
+import argparse
+
+if "ZEPHYR_BASE" not in os.environ:
+    logging.error("$ZEPHYR_BASE environment variable undefined.\n")
+    exit(1)
+
+logger = None
+
+repository_path = os.environ['ZEPHYR_BASE']
+sh_special_args = {
+    '_tty_out': False,
+    '_cwd': repository_path
+}
+
+def init_logs():
+    global logger
+    log_lev = os.environ.get('LOG_LEVEL', None)
+    level = logging.INFO
```

Comment:
I don't know if the same rule applies in python but I would revers these 2 lines, to log_lev gets assigned right before the if that checks it. Or add an empty line before the if. either/or

Unless you find it useless to do any change here :)